web/help/GuideUnit1.md

```diff
@@ -716,60 +746,65 @@ Here is a table of colors you can use:
-    CodeWorld.  To make your own coordinate plane, use this code:
+    CodeWorld.  
+    
+To make your own coordinate plane, use this code
 
     ~~~~~ . clickable
     program = drawingOf(coordinatePlane)
     ~~~~~
 
-This tool combines of two number lines.
+This tool is a combination of two number lines:
+
+1. The first number line is *horizontal*, like the *x-axis*, with positive numbers 
+   to the right and negative numbers to the left.  
+   You can use this line to describe how far left or right to move a picture.
+2. The second number line is *vertical*, like the *y-axis*, with positive numbers 
+   on top, and negative numbers on the bottom.  
+   You use that number line to describe how far up or down to move a picture.
 
-1. The first number line is *horizontal*, with positive numbers to the right
-   and negative numbers to the left.  You can use this line to describe how
-   far left or right to move a picture.
-2. The second number line is *vertical*, with positive numbers on top, and
-   negative numbers on the bottom.  You use that number line to describe
-   how far up or down to move a picture.
+Using combinations of both coordinates together, you can move a picture anywhere you like.  
 
-With both numbers together, you can move a picture anywhere you like.  The
-basic shapes you have learned are drawn with their center at the *origin* --
+The basic shapes you have learned are drawn with their center at the *origin* --
 the point (0, 0) -- on the coordinate plane, so you should measure your x
 and y distances from there.  As you define your own pictures, it's a good
-idea to continue doing the same thing.
+idea to continue this practice.
 
-For example, if you wanted a circle representing the sun in the top left
-corner of the screen.  First, you could look at the horizontal number line,
+For example, suppose you wanted a circle representing the sun in the top left
+corner of the screen.  First, you could look at the horizontal number line
 and see that negative numbers are used to move a shape to the left.  You might
-pick -5, which is five units left on the screen.  Next, you could look at the
-vertical number line, and see that positive numbers are used to move an object
-up.  You might pick 7, which is most of the way to the top of the screen.
-
-The expression describing a picture of the sun in the right place is now
-`translated(sun, -5, 7)`.  The first argument is the preimage, which you
-would define elsewhere in the program.  The second and third arguments are
-the distances to move the shape in your new picture, with horizontal first,
-then vertical.
+pick x = -5, which is five units moving left from the origin on the screen.  
+Next, you could look at the vertical number line and see that positive numbers are 
+used to move an object up.  You might pick 7, which is most of the way 
+to the top of the screen. This should give you a proper placement for the sun in your image.
```

Comment:
I am worried a little but about this paragraph.  The idea we want to get across here is that `translated(sun, -5, 7)` means to shift the sun 5 units to the left, and 7 units up.  It would be WRONG if students thought that `translated(sun, -5, 7)` means to move the sun to the point (-5, 7).

To see the difference, you could consider this code:

    a = translated(circle(5), 2, 2)
    b = translated(a, -2, -2)

Students should be able to reason that `b` is a picture of a circle at the origin, not at (-2, -2).  (Here's why: `b` starts with the preimage `a`, which is a circle centered at (2, 2), and then shifts it 2 units left and 2 units down.  That moves it back to the origin.)

Some students will expect that `b` moves the circle **to** (-2, -2), and that it shouldn't matter where it was before it was moved.  That is, they will mistakenly believe that translation is absolute, instead of relative.  I don't know how to address that misconception here without getting into nesting or combining transformations.  But at the very least, we could make sure this isn't actively giving students the wrong idea.

Reply:
I'll think about a correct strategy for this. That's why it's good to have you here and that you are patient. I have to think about how to word that information. There is a subtlety, well more than that, but definitely a difference.